app/leagues/page.tsx

```diff
@@ -18,36 +21,36 @@ const Leagues = (): JSX.Element => {
         <LeagueCard
           href="/leagues"
           leagueCardLogo="https://ryanfurrer.com/_astro/logo-dark-theme.CS8e9u7V_JfowQ.svg" // should eventually be something like league.logo
-          survivors={69}
+          survivors={gameGroup.survivors.length}
```

Comment:
I don't get it. Isn't this just grabbing 1 league's survivors and participants? It's supposed to grab the actual league's data.

Reply:
I thought the dataStore was what grabbed the actual data and doing this is what displays it.